.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,6 +1,5 @@
 blank_issues_enabled: true
```

Comment:
You'll need to fix the indentation to pass the linting check.

```
blank_issues_enabled: true
contact_links:
  - name: Question
    url: https://stackoverflow.com/questions/tagged/chainlink
    about: Please ask and answer questions here.

```

Reply:
Whoops, thank you. Ran `yarn prettier:run`